index.src.html

```diff
@@ -1477,40 +1465,14 @@
         adjustments.
   </dd>
 
-  <dt><dfn dict-member for=VideoDecoderConfig>cropLeft</dfn></dt>
-  <dd>
-    The number of pixels to remove from the left of the VideoFrame, prior to
-        aspect ratio adjustments. Defaults to zero if not present.
-  </dd>
-
-  <dt><dfn dict-member for=VideoDecoderConfig>cropTop</dfn></dt>
-  <dd>
-    The number of pixels to remove from the top of the VideoFrame, prior to
-        aspect ratio adjustments. Defaults to zero if not present.
-  </dd>
-
-  <dt><dfn dict-member for=VideoDecoderConfig>cropWidth</dfn></dt>
-  <dd>
-    The width in pixels to include in the crop, starting from cropLeft.
-        Defaults to codedWidth if not present.
-  </dd>
-
-  <dt><dfn dict-member for=VideoDecoderConfig>cropHeight</dfn></dt>
-  <dd>
-    The height in pixels to include in the crop, starting from cropLeft.
-        Defaults to codedHeight if not present.
-  </dd>
-
   <dt><dfn dict-member for=VideoDecoderConfig>displayWidth</dfn></dt>
   <dd>
-    Width of the VideoFrame when displayed. Defaults to cropWidth if not
-        present.
+    Width of the VideoFrame when displayed.
```

Comment:
Here and below, I would still add mention of the default, which is now {{VideoFrame/cropWidth}}

Reply:
Note that the interpretation is as an aspect ratio, and the actual display size of a frame is the same as for `<video>` (github.com/whatwg/html/commit/2e94aa64fcf9adbd2f70d8c2aecd192c8678e298)--one dimension of the visible size is scaled up until the ratio is the same as the display size.

Reply:
I added a reference to that spec, and refer to it in a few places.

Reply:
For those following along, see my comment for in the issue
https://github.com/w3c/webcodecs/issues/94#issuecomment-845483687